src/lib.rs

```diff
@@ -1246,7 +1267,7 @@ impl Hasher {
         // also. Convert it directly into an Output. Otherwise, we need to
         // merge subtrees below.
         if self.cv_stack.is_empty() {
-            debug_assert_eq!(self.chunk_state.chunk_counter, 0);
+            // debug_assert_eq!(self.chunk_state.chunk_counter, 0);
```

Comment:
This must be disabled because the chunk counter can be set to a non zero value in the constructor now.